floris/simulation/grid.py

```diff
@@ -73,12 +77,12 @@ class Grid(ABC):
     n_wind_speeds: int = field(init=False)
     n_wind_directions: int = field(init=False)
     turbine_coordinates_array: NDArrayFloat = field(init=False)
-    x: NDArrayFloat = field(init=False, default=[])
-    y: NDArrayFloat = field(init=False, default=[])
-    z: NDArrayFloat = field(init=False, default=[])
     x_sorted: NDArrayFloat = field(init=False)
```

Comment:
Given the added complexity in the grids, this is a good time to document the `_sorted` and `_sorted_inertial_frame` variables. The minimum documentation would here in the docstring, and better would be to also add to the Floris_101 notebook in the docs with detailed exaplanation and code or analysis of these variables.